Index: tenquare_sms/src/main/java/com/zhangyaoxing/sms/listener/SmsListener.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- tenquare_sms/src/main/java/com/zhangyaoxing/sms/listener/SmsListener.java	(revision f8d20aadf41080a79149331016610dfc51a43f7b)
+++ tenquare_sms/src/main/java/com/zhangyaoxing/sms/listener/SmsListener.java	(date 1597998964906)
@@ -11,7 +11,7 @@
 import java.util.Map;
 
 @Component
-@RabbitListener(queues = "sms")//监听的是 sms
+@RabbitListener(queues = "sms")//监听的是 sms  git数据测试
 public class SmsListener {//消息的消费  就是客户  其实可以全写  tensquare_user的  这个是练习为了看的具体就下这里
 
     @Autowired
